File: Nutritional Infomation Calculator/Models/MenuItem.cs
using Nutritional_Infomation_Calculator.Data;
using System.ComponentModel.DataAnnotations;

namespace Nutritional_Infomation_Calculator.Models
{
    /// <summary>
    /// Represents a single menu item
    /// </summary>
    public class MenuItem
    {
        /// <summary>
        /// The menu item's unique id
        /// </summary>
        [Key]
        public int MenuId { get; set; }

        /// <summary>
        /// Name of menu item
        /// </summary>
        public string Title { get; set; }

        /// <summary>
        /// The menu item's image url
        /// </summary>
        public string Image { get; set; }

        /// <summary>
        /// The menu item's nutritional information 
        /// </summary>
        public Nutrition Nutrition { get; set; } = new Nutrition();

        /// <summary>
        /// The menu item's Servings information
        /// </summary>
        public Servings Servings { get; set; } = new Servings();
    }
}

File: Nutritional Infomation Calculator/Models/JavaScriptModels/MenuItem.js

class MenuItem
{
    constructor(menuId, title, image, nutrition, servings) {
        MenuId = menuId;
        Title = title;
        Image = image;
        Nutrition = nutrition;
        Servings = servings;
    }
}

class Nutrient {
    constructor(name, amount, unit, percentOfDailyNeeds) {
        Name = name;
        Amount = amount;
        Unit = unit;
        PercentOfDailyNeeds = percentOfDailyNeeds
    }
}

class Nutrition {
    constructor(nutrients) {
        Nutrients = nutrients;
    }
}

class Servings {
    constructor(number, size, unit) {
        this.Number = number;
        Size = size;
        Unit = unit;
    }
}